interface.py

```diff
@@ -1,19 +1,20 @@
 class InterfaceState:
-    def __init__(self):
+    def __init__(self, change_callback):
         self.dealt_cards = []
         self.robot = None
-        #len(self.program) is the number of cards to select (0 - 5)
         self.program = [None, None, None, None, None]
         self.blocked_cards = []
         self.power_down = False
         self.selection_confirmed = False
         self.cursor_index = 0  # 0-4 number of positon
         self.players = []
         self.game_round = None
-        # List of winners
-        self.winner = []
+        self.winner = None
         self.timer = False
         self.flag_count = 0
+        # Change_callback - callback function which is called with argument,
```

Comment:
Funkce "change_callback" nebere zadne argumenty, kdyz se vola v jednotlivych metodach.
Muj navrh na zdejsi komentar: 
```
# Assign the function that should be called within some InterfaceState methods.
# Assigned function is be called after every robot-related interface state change.
# It is called eg. after choosing or returning cards on hand.
# It is not called on change of the purely visual elements of interface, like moving the cursor.
```

Reply:
to bylo myšleno asi which is called by argument od client interface :-D

Reply:
Zde totez...